dxp/wysiwyg.md

```diff
@@ -0,0 +1,33 @@
+# WYSIWYG Editors
+
+We have recently worked on reducing variability in the WYSIWYG writing experience. After analyzing current usage of the editors the decision was made to provide a more cohesive and comprehensive experience for each specific use case. Documenting the **WYSIWYG** editors that are available in [Liferay DXP](https://github.com/liferay/liferay-portal) can be useful.
+
+At the time of writing this document, there are three different **WYSIWYG** editors in [Liferay DXP](https://github.com/liferay/liferay-portal):
+
+- [AlloyEditor](https://alloyeditor.com)
+- [CKEditor 4](https://ckeditor.com/ckeditor-4/)
+- [TinyMCE](https://www.tiny.cloud/) - disabled by default
```

Comment:
What does "disabled by default" mean?

Reply:
I beleive it means that it's not active by default (or at least once it's moved to the `archived` directory it won't be) - but we can remove this